### ISSUE TITLE
plugins.hparams API missing LogarithmicInterval in options for display

### ISSUE THREAD

**Issue by brethvoice** [tensorflow/tensorboard#4800]
You only have real intervals.  If people know they want a hyper-parameter displayed on that scale, why not make it easier for them?

**Comment by psybuzz** [comment]
Thanks for filing this request!  I could use some help to understand what the desired outcome would look like.

Assuming that the premise of this request is to introduce a `LogarithmicInterval` Hparam (not Metric) class alongside `RealInterval`, `IntInterval`, and `DiscreteInterval`, what would the desired signature of the logarithmic version look like?  In the TensorBoard UI, what sort of unique display treatment would you like to see for it?

**Comment by arghyaganguly** [comment]
@brethvoice , please feel free to reopen this issue with more insights on @psybuzz's comment.
Closing this currently as it has been inactive with awaiting response for 3 days.

**Comment by brethvoice** [comment]
@psybuzz I apologize for the delayed response.  I am new to Github and am not sure how to re-open the issue per suggestion by @arghyaganguly.

The desired behavior would be that in Tensorboard, the radio button for that hyper-parameter in the "Scatter Plot Matrix" view would default to "Logarithmic." Currently the default is "Linear" (the other option is "Quantile" so maybe there needs to be something to make that happen automatically too, but that is not my primary concern).

**Comment by psybuzz** [comment]
Ah, I see now that the "Scatter Plot Matrix" tab has the 'linear' option as the default, thanks for explaining.  If I understand correctly, the downside of this missing API is that manually changing all the radio buttons to "Logarithmic" is not convenient, and especially so if reloading the page resets all the radio buttons back to 'linear'.

Changing the logging API is one option to control this scatter plot axis display type.  Alternatively, if we just made TensorBoard's UI "remember" the previously configured default/logarithmic/quantile settings, users would only have to change the options for each radio button to 'Logarithmic' once.  Would this frontend-only change address the concern?

**Comment by brethvoice** [comment]
@psybuzz I think that would address part of the concern, and possibly the most important part.  As a user I had not even really noticed the fact that resetting the page gets rid of the human-clicked radio button selections.

I only recommended this option because I was digging through the plugins.hparams API and noticed something seemed to be missing.  More users would probably benefit from implementing your suggestion than mine though.
